dev-app/src/screens/PaymentMethodSelectScreen.tsx

```diff
@@ -0,0 +1,94 @@
+import { RouteProp, useNavigation, useRoute } from '@react-navigation/core';
+import React from 'react';
+import { FlatList, StyleSheet, Switch, Text, View } from 'react-native';
+import { colors } from '../colors';
+
+import type { RouteParamList } from '../App';
+import type { IPaymentMethodType } from '../types';
+import ListItem from '../components/ListItem';
+
+export default function PaymentMethodSelectScreen() {
+  const navigation = useNavigation();
+  const { params } =
+    useRoute<RouteProp<RouteParamList, 'PaymentMethodSelect'>>();
+
+  const [pendingPaymentMethodTypes, setPendingPaymentMethodTypes] =
+    React.useState<IPaymentMethodType[]>(params.paymentMethodTypes);
+
+  const doCancel = () => {
+    // Revert any pending changes
+    setPendingPaymentMethodTypes(params.paymentMethodTypes);
+    if (navigation.canGoBack()) {
+      navigation.goBack();
+    }
+  };
+
+  return (
+    <View style={styles.container}>
+      <Text style={styles.info}>
+        Please select eligible payment methods for this payment:
+      </Text>
+      <ListItem
+        title="Confirm"
+        testID="confirm-button"
+        color={colors.green}
+        onPress={async () => {
+          params.onChange(pendingPaymentMethodTypes);
+          if (navigation.canGoBack()) {
+            navigation.goBack();
+          }
+        }}
+        disabled={
+          pendingPaymentMethodTypes.filter((pmt) => pmt.enabled).length === 0
+        }
+      />
+      <ListItem
+        title="Cancel"
+        testID="cancel-button"
+        color={colors.red}
+        onPress={doCancel}
+      />
+      <FlatList
```

Comment:
FlatList seems have issue when using with Android simulator, so consider change to ScrollView.

ref: https://github.com/stripe/stripe-terminal-react-native/pull/829